src/components/UserIcon/UserIcon.module.css

```diff
@@ -0,0 +1,23 @@
+.userIcon {
+  display: block;
+  width: 2.5rem;
+  height: 2.5rem;
+  margin: 0.5rem;
+}
+
+.userIcon::before {
+  content: attr(data-user-initials);
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 2.5rem;
```

Comment:
The width and height for the absolute positioned element should be `100%`, so we can change the sizes by changing the parent values 😊